extensions/extensions-helm/extensions-helm-pipeline-invocation/aissemble-pipeline-invocation-app-chart/values.template.yaml

```diff
@@ -32,11 +33,13 @@ aissemble-quarkus-chart:
       - mp.messaging.incoming.pipeline-invocation.value.deserializer=org.apache.kafka.common.serialization.StringDeserializer
 
   service:
-    spec:
-      ports:
-        - name: "quarkus-management"
-          port: 9001
-          targetPort: 9000
+    ports:
+      - name: "invocation-rest"
+        port: 8085
+        targetPort: 8080
+      - name: "quarkus-mgmt"
```

Comment:
I: Hit the 15 character limit (since this was previously incorrectly configured the length error was never detected)